library/script/usdot_airports.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+
+from . import df_to_tempfile
+
+
+class Scriptor:
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+    def ingest(self) -> pd.DataFrame:
+        df = pd.read_excel('library/tmp/all-airport-data.xlsx', sheet_name='Airports')
```

Comment:
@AmandaDoyle @SashaWeinstein thoughts on reading in all the airport data - should we filter to the nyc subset of the data?

Reply:
If there's an easy way to filter to just New York City or New York State, and it increases performance, then yes; otherwise I wouldn't worry about it.